binding.cc

```diff
@@ -1,62 +1,65 @@
-#include <node.h>
-#include <v8.h>
+#include <nan.h>
 
 #include <bcm_host.h>
 
-using namespace v8;
-
 static VCHI_INSTANCE_T vchi;
 static VCHI_CONNECTION_T *vchi_connections;
 
 static char buf[1024];
 
-Handle<Value> Request(const Arguments& args) {
-    HandleScope scope;
+NAN_METHOD(Request) {
+    if (info.Length() < 1) {
+        Nan::ThrowTypeError("Wrong number of arguments");
+        return;
+    }
+    if (!info[0]->IsString()) {
+        Nan::ThrowTypeError("First argument must be a string");
+        return;
+    }
 
-    String::AsciiValue str(args[0]);
+    v8::String::Utf8Value str(info[0]);
```

Comment:
v8::String::AsciiValue seems to be deprecated according to https://strongloop.com/strongblog/node-js-v0-12-c-apis-breaking/ "Removal of v8::String::AsciiValue" ...

I don't know if Utf8Value is the best choice here, but as the commands are all ASCII, it doesn't matter.